extensions/applicationinsights-analytics-js/src/JavaScriptSDK/AnalyticsPlugin.ts

```diff
@@ -738,16 +670,13 @@ export class ApplicationInsights extends BaseTelemetryPlugin implements IAppInsi
                         return true;
                     }
 
-                    _addTelemetryInitializer(dropBrowserLinkRequests)
+                    _self.addTelemetryInitializer(dropBrowserLinkRequests);
+                    _browserLinkInitializerAdded = true;
                 }
             }
 
-            function _addTelemetryInitializer(telemetryInitializer: (envelope: ITelemetryItem) => boolean | void) {
-                _self._telemetryInitializers.push(telemetryInitializer);
-            }
-
             function _sendCORSException(exception: IAutoExceptionTelemetry, properties?: ICustomProperties) {
-                const telemetryItem: ITelemetryItem = TelemetryItemCreator.create<IAutoExceptionTelemetry>(
+                let telemetryItem: ITelemetryItem = TelemetryItemCreator.create<IAutoExceptionTelemetry>(
```

Comment:
Note: changing to let as some frameworks that use es6 will freeze any objects defined as `const`